site/documentation/content/concepts/device-provisioning/index.md

```diff
@@ -83,3 +81,33 @@ The newly created credentials are returned to the protocol adapter in the respon
 The following query of the Device Registration API returns the previously generated registration data.
 
 The provisioning is, of course, a one-time action, on subsequent connections the APIs simply return the stored records.
+
+#### Gateway based Auto-Provisioning
+
+Enabling gateway based auto-provisioning requires configuration of the gateway device:
+
+##### Step 1: Configure the Gateway Device
+
+The gateway device must have the corresponding authority (`auto-provisioning-enabled`) set.
+Refer to the [Management API]({{< ref "/api/management" >}}) for creating or updating a 
+corresponding device.
+
+##### Step 2: Connect an unregistered Device via Gateway to Hono
+
+{{< figure src="auto-provisioning-gateway.svg" alt="A unregistered device connects to a gateway which uses the 
+Device Registration API to provision it" title="Automatic Provisioning of a Device via gateway" >}}
+
+The yet unregistered edge device sends telemetry data via a gateway (1) to a protocol adapter which then checks if the 
+edge device is already registered by calling the "assert" operation of Device Registration API (2). If the device 
+registration service finds that the device isn't registered yet and the gateway has the `auto-provisioning-enabled` 
+authority set, it creates the edge device (3). 
+
+Subsequently, after it made sure that it hasn't already done so, it sends an empty event 
+(see [Event API]({{< ref "/api/event" >}})) with the `hono_registration_status` application property being set to `NEW` 
+to the AMQP network (4). Upon receiving the disposition for the sent event (5), it marks the event as having been 
+delivered (6) in order to prevent sending duplicate events 
```

Comment:
this is not so much about avoiding duplicates but about making sure that we send the event at least once, e.g. when the registration service has crashed before it could mark the event as delivered, right?

Reply:
Yes, I rephrased this as it is more about making sure the Empty Notification is sent at least once. A crash of the registration service in the very moment it starts to send this event would be one possible case being mitigated by the persistent flag as it would re-send the notification when the next telemetry is received.